### PR TITLE
[BACKLOG-30864] Fallback to legacy shim configuration if no config fi…

### PR DESCRIPTION
**Pull request by peterrinehart**
…les found in metastore locations.

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 13s
Build command:
```console
mvn clean deploy -B -e -Daudit -pl \
common-fragment-V1,common-services-api
```
---
:ok_hand: All tests passed!
 
Tests run: **766**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/big-data-hadoop-shims/job/PR-1044/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 11s
Build command:
```console
mvn clean deploy -B -e -Daudit -pl \
common-fragment-V1,common-services-api
```
---
:ok_hand: All tests passed!
 
Tests run: **766**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/big-data-hadoop-shims/job/PR-1044/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: common-fragment-V1/pom.xml [hunk 1]
```diff
@@ -550,7 +550,13 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>org.pentaho</groupId>
+      <groupId>pentaho</groupId>
```
Comment (jgminder) [inline common-fragment-V1/pom.xml]: Let's clean up the spacing in the pom.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 20s
Build command:
```console
mvn clean deploy -B -e -Daudit -pl \
common-fragment-V1,common-services-api
```
---
:ok_hand: All tests passed!
 
Tests run: **766**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/big-data-hadoop-shims/job/PR-1044/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**